jdaviz/configs/mosviz/plugins/parsers.py

```diff
@@ -50,6 +51,9 @@ def _add_to_table(app, data, comp_label, table_viewer_reference_name='table-view
     if data is None or len(data) == 0:
         return
 
+    # Convert any nans to strings for JSON serialization
+    data = [str(x) if pd.isna(x) else x for x in data]
```

Comment:
Seems a little overkill to introduce pandas as required dependency just to use `isna`.

Reply:
That was my concern as well, but the only way to introduce this check that I could reasonably find. I wanted to gather thoughts as to whether this is something we want or not.

The plus is that pandas is already an indirect dependency 🤷

Reply:
You mentioned that `np.isnan` does not support complex object, but is this RA/Dec problem that complex?

Reply:
> pandas is already an indirect dependency

Huh, which packaging is pulling it in?

Regardless, before this patch, Jdaviz might still work if someone do a `pip uninstall pandas`, but that is no longer the case.

Reply:
> is this RA/Dec problem that complex?

This isn't necessarily only for the RA/Dec; this is for any element that the user wants to add to the table. This came up because we add the actual Glue Data objects for each row to the table itself in hidden columns ("2D Spectrum", "1D Spectrum", "Image"). So `np.isnan` was failing when it was trying to inspect the Glue Data objects

Reply:
Removed! See https://github.com/spacetelescope/jdaviz/pull/1958#issuecomment-1403934536